frontend/models/product-list/index.ts

```diff
@@ -50,66 +50,81 @@ export async function findProductList(
    const result = await strapi.getProductList({
       filters,
    });
+
    const productList = result.productLists?.data?.[0]?.attributes;
-   if (productList == null) {
+
+   const deviceTitle =
+      productList?.deviceTitle ?? filters.deviceTitle?.eq ?? '';
+   const deviceWiki = deviceTitle ? await fetchDeviceWiki(deviceTitle) : null;
+
+   if (productList == null && deviceWiki == null) {
       return null;
    }
-   const productListImageAttributes = productList.image?.data?.attributes;
 
-   const deviceWiki = productList.deviceTitle
-      ? await fetchDeviceWiki(productList.deviceTitle)
+   const handle = productList?.handle ?? '';
+   const parents =
+      productList?.parent ??
+      (deviceWiki?.ancestors
+         ? convertAncestorsToStrapiFormat(deviceWiki.ancestors)
+         : null);
+   const type = productList?.type ?? deviceWiki?.namespace;
+   const title =
+      productList?.title ??
+      (deviceWiki?.title ? deviceWiki?.title + ' Parts' : '');
+   const description =
+      productList?.description ?? deviceWiki?.description ?? '';
+   const productListImageAttributes = productList?.image?.data?.attributes;
+   const image = productListImageAttributes
```

Comment:
IIRC we aren't showing the main image at all anymore (only child images) so we can probably lose this.